src/figuresClasses.ts

```diff
@@ -1,11 +1,88 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
 
-export class Triangle implements Figure {}
+  color: string;
 
-export class Circle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  constructor(
+    color: string,
+    public a: number = 0,
+    public b: number = 0,
+    public c: number = 0,
+  ) {
+    if (a + b <= c || c + b <= a || a + c <= b) {
+      throw new Error('its not triangle');
+    }
+    this.color = color;
+    this.shape = 'triangle';
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
```

Comment:
The 'getArea' function should not accept any parameters according to the checklist. All necessary values should be accessible as object fields, which they are in this case. So, the function signature is correct as per checklist item 1.